pkg/apis/core/validation/validation.go

```diff
@@ -2824,6 +2824,17 @@ func validateDNSPolicy(dnsPolicy *core.DNSPolicy, fldPath *field.Path) field.Err
 	return allErrors
 }
 
+func validateFSGroupChangePolicy(fsGroupPolicy *core.PodFSGroupChangePolicy, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	validValues := sets.NewString(string(core.AlwaysChangeVolumePermission), string(core.OnRootMismatch))
```

Comment:
minor: file-scope means you only allocate it once.

Reply:
fixed